pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -8,31 +8,29 @@ import (
 	"strings"
 	"time"
 
-	"k8s.io/apimachinery/pkg/util/sets"
-
+	securityv1 "github.com/openshift/api/security/v1"
```

Comment:
> Please move back to the previous order.

> It's odd that your IDE is mixing all imports together at all times eyes

What value have you gotten value from separating beyond standard library and everything else?  The auto-importers tend to work better with a single region.  We did this originally in kube because auto-importers didn't exist for golang, but it has stuck around really annoyingly.